daemon/stats/collector_linux.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build !darwin !windows
```

Comment:
You can revert this update to the original, and add stub/empty function of `system.GetClockTicks()` (originally from github.com/opencontainers/runc/libcontainer/systes) to minimize the patches.

Reply:
If we revert this commit back, we end us with these error: 

1) undefined: platformNewStatsCollector
2) s.getNumberOnlineCPUs undefined (type *Collector has no field or method getNumberOnlineCPUs)

Reply:
This is done, with the addition of a new dep(github.com/tklauser/go-sysconf) to get clock ticks.

Reply:
pls pin-point which commit includes this modification (it's hard to look from huge diffs).

Reply:
https://github.com/Aniketh01/engine/commit/f2a886fcc79d38bb792177f4b3ca5c7bc73aac05